recipes-sca/checkbashism-native/nativesdk-checkbashism_2.23.6.bb

```diff
@@ -0,0 +1,34 @@
+SUMMARY = "check for bashisms in /bin/sh scripts"
```

Comment:
IIRC there's no support for nativesdk tooling in kirkstone, so I guess it's better to drop this file here

Reply:
When I cherry-picked from master I did get a merge conflict due to the nativesdk-checkbashism_2.23.6.bb not previously existing, so I accepted adding it as an new file.  I can remove this file from the kirkstone commit if that is your preference?

Reply:
I removed  the new file **nativesdk**-checkbashism_2.33.6.bb from the PR